src/routes/Review/components/AboutYourProject/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import HelpBanner from "components/HelpBanner";
+import PageUl from "../../../../components/PageElements/PageUl";
+
+const AboutYourProject = () => {
+  return (
+    <HelpBanner
+      styles={["turqoise", "marginTop24"]}
+      title="Important things to know about your project"
+    >
+      <PageUl>
+        <li>
+          <strong>
+            Your Dashboard is your go-to hub for managing your work.
+          </strong>
+          &nbsp; From here you can view timelines, details, and a more important
```

Comment:
This would read better as:

`From here you can view timelines, details, and other important`

Replace "a more" with "other".

Reply:
Done